sql1.txt

```diff
@@ -20,13 +20,10 @@ WHERE Price < 20;
 
 3. Retrieve all employees whose last name starts with C.  Paste your SQL statement below:
 
-[["EmployeeID", "LastName", "FirstName", "BirthDate", "Photo", "Notes"],
- [8,
-  "Callahan ",
-  "Laura ",
-  "1/9/1958 ",
-  "EmpID8.pic ",
-  "Laura received a BA in psychology from the University of Washington. She has also completed a course in business French. She reads and writes French. "]]
+SELECT ProductName, Price
+FROM Products
+WHERE Price < 20;
```

Comment:
Thank you for implementing the changes. Please modify this code to retrieve all employees whose last name starts with 'C'. After that, I will approve your project."

Reply:
Thank you for review , done 👍 
<img width="409" alt="image" src="https://github.com/DYA13/sqlintro/assets/88336329/75f58da9-ae43-4832-94da-2a63a0031b9b">